lib/customer.rb

```diff
@@ -0,0 +1,47 @@
+class Customer
+
+  attr_accessor :order, :total_price, :total_quantity
+  attr_reader :summary
+
+  def initialize
+    @order = {}
+    @total_price = 0
+    @total_quantity = 0
+    # @summary = [["Item","Quantity","Price"]]
+  end
+
+  def check_menu(menu)
+    menu.view_menu
+  end
+
+  def add_to_order(menu,food,quantity)
+    self.order.has_key?(food.to_sym) ? self.order[food.to_sym][0] += quantity : order[food.to_sym] = [quantity, menu.items[food.to_sym]]
+    self.calculate_total_price
+    self.calculate_total_quantity
+  end
+
+  def calculate_total_price
+    self.total_price = 0
+    order.each {|k,v| self.total_price += v[0]*v[1] }
+  end
+
+  def calculate_total_quantity
+    self.total_quantity = 0
+    order.each {|k,v| self.total_quantity += v[0] }
```

Comment:
Unused block argument - `k`. If it's necessary, use `_` or `_k` as an argument name to indicate that it won't be used.<br>Space missing after comma.<br>Space between { and | missing.